website/docs/configuration/securing-access-to-the-dashboard.mdx

```diff
@@ -52,7 +52,7 @@ kubectl create secret generic oidc-auth \
   --from-literal=tokenDuration=<token-duration>
 ```
 
-Once the HTTP server starts, it will redirect unauthenticated users to the provider's login page to authenticate them. Upon successful authentication, the users' identity will be impersonated in any calls made to the Kubernetes API, as part of any action they take in the dashboard. At this point, the dashboard will fail to render correctly unless RBAC has been configured accordingly. Follow the instructions in the [RBAC authorization](#rbac-authorization) section in order to configure RBAC correctly.
+Once the HTTP server starts, it will redirect unauthenticated users to the provider's login page to authenticate them. Upon successful authentication, the users' identity will be impersonated in any calls made to the Kubernetes API, as part of any action they take in the dashboard. By default the Helm chart will configure RBAC correctly but it is recommended to read the [service account](docs/configuration/service-account-permissions) and [user](docs/configuration/user-permissions) permissions pages to understand which actions are needed for Weave GitOps to function correctly.
```

Comment:
> Once the HTTP server starts, it will redirect unauthenticated users to the provider's login page to authenticate them.
I'm not sure if it's true. It will redirect to the login page of the weave-gitops UI. There they can click on the OIDC login button and they will be redirected to the provider's page. Or is there a case when the built-in login page is skipped automatically?